app/src/utils/data/ipfs.ts

```diff
@@ -6,7 +6,7 @@ import { BigNumber } from 'src/utils/ethers';
 import { LocalForageData } from 'src/types';
 import { Ref } from 'vue';
 
-const retrievalEndpoint = 'https://ipfs.fleek.co/ipfs';
+const retrievalEndpoint = 'https://scopelift.b-cdn.net/ipfs/<cid>/';
```

Comment:
I don't think we should be including `<cid>` in the endpoint:

```suggestion
const retrievalEndpoint = 'https://scopelift.b-cdn.net/ipfs';
```

(
[https://scopelift.b-cdn.net/ipfs/<cid\>/QmTQMgoxDRj8gfNn5Cvznt5CoEE6cz9MQeZrQsNb36BMdm ](https://scopelift.b-cdn.net/ipfs/<cid>/QmTQMgoxDRj8gfNn5Cvznt5CoEE6cz9MQeZrQsNb36BMdm 
)
vs 
https://scopelift.b-cdn.net/ipfs/QmTQMgoxDRj8gfNn5Cvznt5CoEE6cz9MQeZrQsNb36BMdm
)

@apbendi - I see in the issue you spoke about the trailing slash but it wasn't there previously - do we need it?

Reply:
so, can it be `https://scopelift.b-cdn.net/ipfs`

Reply:
I think that should work - the URL above resolves so I don't think we need that trailing slash